lib/public_key/src/public_key.erl

```diff
@@ -587,6 +644,12 @@ generate_key({rsa, ModulusSize, PublicExponent}) ->
                              when OthersECDHkey :: #'ECPoint'{},
                                   MyECDHkey :: #'ECPrivateKey'{},
                                   SharedSecret :: binary().
+compute_key(#'ECPoint'{point = Point}, #'ECPrivateKey'{privateKey = PrivKey,
+						       parameters = {namedCurve, Curve} = Param})
+  when (Curve == 'id-X25519') orelse
+       (Curve == 'id-X448') ->
```

Comment:
When is 'id-X448' used and when is ?'id-X448' to be used? What is the distinction? Is this really correct here , to not use ?'id-X448'  ??

Reply:
Good catch this is a mistake! I will fix it.